### DISCUSSION TITLE
Training Fastconformer-CTC on Kazakh Language

### DISCUSSION THREAD

**Discussion by diasbalmash** [NVIDIA-NeMo/NeMo#9506] [Q&A]
Hello, community. I just started working with NeMo. So, I want to train ASR based on Fastconformer-CTC with 100 hours of recorded calls in kazakh language. What type of training will be better for my situation, fine-tuning some pre-trained model or train it from scratch? Also can someone give info about the data processing and manifest?

**Comment by titu1994** [comment]
We've found that almost without exception, starting from a pertained checkpoint, no matter the language will do better than starting from scratch. 

You can get better scores of the pretraining language is similar to the new language, but usually not significantly different than starting with a model trained on thousands of hours of English

**Reply by diasbalmash** [reply]
Can you give the info about the data processing and data manifest format?
